bin/bb8/status.py

```diff
@@ -45,7 +45,7 @@ def last_modified_remote(fm: RemoteFileManager, paths: RemotePaths):
 
 
 def last_modified_local(target, docker_client):
-    cmd = 'find -L /data -type f -print0' \
+    cmd = 'find -L /data -type f -o -type d -print0' \
```

Comment:
I think this command will have the same issue/also needs the `-print0` flag moving

Reply:
Yes, you're right. For some reason I thought the version of `find` inside the rsync container was different, and it would be fine. But I've tested it and found the same problem, and I've confirmed the problem goes away once I make the same change.